source/_docs/configuration/packages.markdown

```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
+
+If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See [Issue 16441](https://github.com/home-assistant/core/issues/16441) and the general documentation for [Authentication Providers](https://www.home-assistant.io/docs/authentication/providers/#configuring-auth-providers).
```

Comment:
We should not send end-users into GitHub issues for information. If information from that linked issue is relevant, please add it to the docs and remove the issue link.

Reply:
removed,, link to auth-providers-doc is enough